packages/rfw/lib/src/dart/binary.dart

```diff
@@ -264,6 +276,10 @@ RemoteWidgetLibrary decodeLibraryBlob(Uint8List bytes) {
 // endianess used by this format
 const Endian _blobEndian = Endian.little;
 
+// whether we can use 64 bit APIs on this platform
+// (on JS, we can only use 32 bit APIs and integers only go up to ~2^53)
+const bool _kHas64Bits = 0x1000000000000000 + 1 != 0x1000000000000000; // 2^60
```

Comment:
Nit: Effective Dart says [not to use `k`](https://dart.dev/effective-dart/style#dont-use-prefix-letters), and Flutter style only says [to use it for global constants](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#minimize-the-visibility-scope-of-constants)

(I would strongly vote for changing the Flutter style guide to remove the `_k...` example and change "It’s not necessary to add the k prefix to non-global constants." to explicitly forbid it, to reduce pointless diff from Effective Dart.)

Reply:
Fixed.

(go for it!)